suite/suite.go

```diff
@@ -63,6 +63,20 @@ func failOnPanic(t *testing.T) {
 	}
 }
 
+// Run provides suite functionality around golang subtests.  It should be
+// called in place of t.Run(name, func(t *testing.T)) in test suite code
+// to expose setup and teardown functionality for each subtest.  The passed-in
```

Comment:
> to expose setup and teardown functionality for each subtest.

This comment is confusing to me. Based on its wording, I would expect some type of before and after method is ran by the suite runner before and after each subtest (similar to the `SetupSuite` and `TearDownSuite` ran before and after the suite itself, or the `SetupTest`, `BeforeTest`, `AfterTest`, and `TearDownTest` ran before and after each test). I don't believe that is the intent though based on the code, and that the intent is to allow people to write subtests in which they may themselves execute setup and teardown code within their passed in function.

To address, I suggest altering this statement to something along the lines of

>  It may be called in place of t.Run(name, func(t *testing.T)) within tests inside a test suite.